exec_syscall.c

```diff
@@ -0,0 +1,13 @@
+#include <common.h>
+#include <exec.h>
+
+int main() {
```

Comment:
I didn't do it just yet, because there indeed seems to be some problem with passing arguments to kernel. It appears to me as if OVPsim ignored `--argv` flag. I know @mkaim intended to investigate this today, so maybe he knows more defails.